src/components/Dropdown/components/option/option.jsx

```diff
@@ -10,15 +10,17 @@ const Option = ({ Renderer, data, children, optionWrapperClassName, ...props })
   const rendererProps = { children, data, ...props };
   return (
     <Tooltip {...tooltipProps} position={Tooltip.positions.RIGHT}>
-      {Renderer ? (
-        <components.Option {...rendererProps} className={cx(styles.optionReset, optionWrapperClassName)}>
-          <Renderer {...rendererProps} {...data} /> {/* Spreading data here for a backward compatability */}
-        </components.Option>
-      ) : (
-        <components.Option {...rendererProps} className={cx(styles.optionReset, optionWrapperClassName)}>
-          <ChildrenContent data={data}>{children}</ChildrenContent>
-        </components.Option>
-      )}
+      <div role="option" aria-selected={props.isSelected}>
```

Comment:
Same comment about react-select, this is a breaking change

Reply:
Same comment, this is how to use it in the option component: 
<img width="483" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/33456543/bf5261fa-3a9d-43af-bf5d-f64d6af83e01">